sunpy/timeseries/sources/eve.py

```diff
@@ -57,35 +57,63 @@ class ESPTimeSeries(GenericTimeSeries):
 
     _source = 'esp'
 
-    @peek_show
-    def peek(self, title='EVE/ESP Level1', **kwargs):
+    def plot(self, axes=None, title="EVE/ESP Level 1", **kwargs):
         """
+        Plots the EVE ESP Level 1 timeseries data.
+
         Parameters
         ----------
+        axes : `matplotlib.axes.Axes`, optional
+            The axes on which to plot the TimeSeries. Defaults to current axes.
         title : `str`, optional
             Plot title.
         **kwargs : `dict`
             Additional plot keyword arguments that are handed to `pandas.DataFrame.plot`.
+
+        Returns
+        -------
+        `~matplotlib.axes.Axes`
+            The plot axes.
         """
 
         self._validate_data_for_plotting()
 
-        names = ('Flux \n 0.1-7nm', 'Flux \n 18nm', 'Flux \n 26nm', 'Flux \n 30nm', 'Flux \n 36nm')
+        if axes is None:
+            axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
 
-        axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
+        names = ("Flux \n 0.1-7nm", "Flux \n 18nm", "Flux \n 26nm", "Flux \n 30nm", "Flux \n 36nm")
 
         axes[-1].set_xlim(self.to_dataframe().index[0], self.to_dataframe().index[-1])
 
         axes[0].set_title(title)
         for i, ax in enumerate(axes):
             ax.set_ylabel(names[i])
-            ax.legend(loc='upper right')
-        axes[-1].set_xlabel('Time (UT) ' + str(self.to_dataframe().index[0])[0:11])
-        axes[-1].xaxis.set_major_formatter(dates.DateFormatter('%H:%M'))
+            ax.legend(loc="upper right")
+        axes[-1].set_xlabel("Time (UT) " + str(self.to_dataframe().index[0])[0:11])
+        axes[-1].xaxis.set_major_formatter(dates.DateFormatter("%H:%M"))
+
+        return axes
 
+    @peek_show
+    def peek(self, title="EVE/ESP Level1", **kwargs):
+        """
+        Plots the GBM lightcurve TimeSeries by calling
+        `~sunpy.timeseries.sources.eve.ESPTimeSeries.plot`.
+
+        Parameters
+        ----------
+        title : `str`, optional
+            Plot title.
+        **kwargs : `dict`
+            Additional plot keyword arguments that are handed to `pandas.DataFrame.plot`.
+        """
+        self._validate_data_for_plotting()
+        axes = self.to_dataframe().plot(subplots=True, sharex=True, **kwargs)
```

Comment:
This are in the plot function, so I don't see why we can't remove them here?

Reply:
Sorry, must've forgotten